packages/client/src/object/aggregate.test.ts

```diff
@@ -398,6 +425,202 @@ describe("aggregate", () => {
     });
   });
 
+  it("works with $orderBy (no groups)", async () => {
+    const mockFetch: Mock = vi.fn();
```

Comment:
You may want to throw this boilerplate setup code for the fetch in a `beforeAll`